docs/releases.md

```diff
@@ -1,5 +1,26 @@
 # Releases
 
+## 1.7
+[Full Release Notes](https://github.com/securefederatedai/openfl/releases/tag/v1.7)
+
+### New Features
+- [**FederatedRuntime**](https://openfl.readthedocs.io/en/latest/about/features_index/workflowinterface.html#runtimes-future-plans) for Workflow API: enables a seamless transition from a local simulation (via LocalRuntime) to a distributed Federated Learning deployment. Check out the [FederatedRuntime 101 Tutorial](https://github.com/securefederatedai/openfl/tree/develop/openfl-tutorials/experimental/workflow/FederatedRuntime/101_MNIST) to try it yourself.

```

Comment:
We should add the following text to the description (this is the outcome of the security review):
 
"The initial version of the FederatedRuntime included in this release is an [experimental feature](https://openfl.readthedocs.io/en/latest/developer_guide/experimental_features.html) that should be used only in an internal environment. We further recommend that users operate only on artificial or public data that is not considered intellectual property. The experimental tag and restrictions will be removed in future releases of OpenFL."